charts/retool/templates/deployment_telemetry.yaml

```diff
@@ -175,7 +175,7 @@ metadata:
     {{- include "retool.telemetry.labels" . | nindent 4 }}
 spec:
   ports:
-    - name: statsd-udp
```

Comment:
Not needed, but just doing for consistency 

> Non-TCP based protocols, such as UDP, are not proxied. These protocols will continue to function as normal, without any interception by the Istio proxy but cannot be used in proxy-only components such as ingress or egress gateways.

https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/